packages/contracts/contracts/L1/messaging/L1CrossDomainMessenger.sol

```diff
@@ -395,7 +395,7 @@ contract L1CrossDomainMessenger is
         L2MessageInclusionProof memory _proof
     )
         internal
-        view
+        pure
```

Comment:
Interesting, does this contribute to the gas savings?

Reply:
I haven't measured the savings per commit but I doubt this saves any gas. It was just a warning fix I snuck in.